keyboards/handwired/horrortroll/k552/rules.mk

```diff
@@ -0,0 +1,36 @@
+# MCU name
+MCU = STM32F103
+
+# GENERIC STM32F103C8T6 board - mass storage bootloader
+MCU_LDSCRIPT = k552_f103
+BOARD = k552
+
+BOOTLOADER = stm32duino
+
```

Comment:
```suggestion
```
If this board has a mass storage bootloader, this isn't correct.

Reply:
This board is STM32F103, it didn’t come with mass storage bootloader by default anyway.

Reply:
In that case I would think the board files and ldscript can be deleted?

Reply:
Nope, since this is RCT6. It needed those board files and ldscript to working